underlay/src/main/resources/config/display/sd/criteriaselector/measurement/startDateGroupByCount.json

```diff
@@ -1,4 +1,11 @@
 {
   "groupByCount": true,
-  "attribute": "start_date"
-}
\ No newline at end of file
+  "attributes": {
+    "observationOccurrence": {
+      "values": [
+        "start_date",
+        "observation"
```

Comment:
`measurementOccurrence` and `measurement`?

Reply:
Good catch.